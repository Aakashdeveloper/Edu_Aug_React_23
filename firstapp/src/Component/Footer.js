import React from 'react';

const Footer = (props) => {
    console.log("in footer>>",props)
    return(
        <React.Fragment>
            <hr/>
            <center>
                &copy; Developer Funnel {props.month} {props.year}
            </center>
        </React.Fragment>
    )
}

export default Footer