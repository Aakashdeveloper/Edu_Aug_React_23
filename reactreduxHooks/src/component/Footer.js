import React from 'react';

const Footer = () => {
    return(
        <footer>
            <div>
                <hr/>
                &copy; Developer Funnels
            </div>
        </footer>
    )
}

export default Footer;