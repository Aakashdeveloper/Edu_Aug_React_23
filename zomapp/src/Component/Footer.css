/* Footer Css */
footer{
    margin-top:2%;
    text-align:center;
    background-color: #3b2632;
    color:white
}
ul{
    list-style: none;
}

.footdiv{
    width:33%;
    float: left;
    border-right: 1px solid white;
}

.noBorder{
    border:none
}

.socialFoot{
    text-align: center;
}


@media screen and (max-width:1600px){
    body{
        background-color:rgb(247, 197, 204)
    }
}

@media screen and (max-width:1400px){
    body{
        background-color:rgb(199, 235, 240)
    }
}

@media screen and (max-width:1200px){
    body{
        background-color:rgb(193, 236, 210)
    }
}

@media screen and (max-width:1000px){
    body{
        background-color:rgb(244, 210, 199)
    }
    .tileContainer{
        width:45%
    }
}

@media screen and (max-width:800px){
    body{
        background-color:rgb(243, 242, 200)
    }
}

@media screen and (max-width:600px){
    body{
        background-color:rgb(195, 219, 245)
    }
    .tileContainer{
        width:90%
    }
}