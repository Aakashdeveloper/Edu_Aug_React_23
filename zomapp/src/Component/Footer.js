import React from 'react';
import './Footer.css';

const Footer = () => { 
    return(
        <footer>
            <hr/>
            <h2>&copy; Developer Funnel</h2>
            <div className="footdiv">
                <ul>
                    <li>About us</li>
                    <li>Contact us</li>
                </ul>
            </div>
            <div className="footdiv">
                <ul>
                    <li>About us</li>
                    <li>Contact us</li>
                </ul>
            </div>
            <div className="footdiv noBorder">
                <ul>
                    <li>About us</li>
                    <li>Contact us</li>
                </ul>
            </div>
            <div id="socialFoot">
                <a href="#">
                    <img src="https://i.ibb.co/TTp516h/facebook.png" alt="fb"
                    className="socialIcon"/>
                </a>
                <a href="#">
                    <img src="https://i.ibb.co/w0kZ5Hf/insta.png" alt="fb"
                    className="socialIcon"/>
                </a>
                <a href="#">
                    <img src="https://i.ibb.co/4jxsYSP/yt3.png" alt="fb"
                    className="socialIcon"/>
                </a>
            </div>
        </footer>
    )
}

export default Footer;