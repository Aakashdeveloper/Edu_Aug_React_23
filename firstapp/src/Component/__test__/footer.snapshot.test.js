import React from 'react';
import Footer from '../Footer';
import {create} from 'react-test-renderer';

describe('Snapshot test',() => {
    test('Testing Footer',() => {
        let tree = create(<Footer/>);
        expect(tree.toJSON()).toMatchSnapshot()
    })
})