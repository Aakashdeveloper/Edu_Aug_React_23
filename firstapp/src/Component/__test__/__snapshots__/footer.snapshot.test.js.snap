// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Snapshot test Testing Footer 1`] = `
Array [
  <hr />,
  <center>
    Â© Developer Funnels 
     
  </center>,
]
`;
